HerPublicWebsite.UnitTests/HerPublicWebsite.UnitTests.csproj

```diff
@@ -12,6 +12,7 @@
         <PackageReference Include="NUnit" Version="3.13.2" />
         <PackageReference Include="NUnit3TestAdapter" Version="4.0.0" />
         <PackageReference Include="coverlet.collector" Version="3.1.0" />
+        <PackageReference Include="RichardSzalay.MockHttp" Version="6.0.0" />
```

Comment:
This looks fine.

Just in case it's not obvious, please do check licences on any open source libraries that you add to the project. Especially any that will be used on the live site, we will need to include a list of the ones we've used on final report.